ekncontent/ekncontent/eknc-object-model.h

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2016 Endless Mobile, Inc. */
+
+#pragma once
+
+#include "eknc-content-object-model.h"
+
+G_BEGIN_DECLS
+
+EkncContentObjectModel *
+eknc_object_model_from_string (const gchar *str,
```

Comment:
Consider making a companion function to create one from a Json node (or even instead of the from-string function)